docs/tutorial.rst

```diff
@@ -155,7 +155,7 @@ argument accepted by all array creation functions. For example::
     >>> from numcodecs import Blosc
     >>> compressor = Blosc(cname='zstd', clevel=3, shuffle=Blosc.BITSHUFFLE)
     >>> data = np.arange(100000000, dtype='i4').reshape(10000, 10000)
-    >>> z = zarr.array(data, chunks=(1000, 1000), compressor=compressor)
+    >>> z = zarr.array(data, chunks=(1000, 1000), compressor=compressor, zarr_format=2)
```

Comment:
This example works for `zarr_format=2` but requires https://github.com/zarr-developers/zarr-python/issues/2398 to work for `zarr_format=3`